plugins/ingest-attachment/src/test/java/org/elasticsearch/ingest/attachment/AttachmentProcessorTests.java

```diff
@@ -284,7 +284,6 @@ public void testEpubDocument() throws Exception {
                 "content_type",
                 "content_length",
                 "date",
-                "keywords",
```

Comment:
Is the fact that we don't generate this now a breaking change?

Reply:
I think that's certainly debatable. We're still looking for it, but tika is no longer producing it for epub files (at least not for this one). Tika is still producing it for other file types. Their documentations mentions that they've removed the `Keywords` metadata field in favor of other ones (https://cwiki.apache.org/confluence/display/TIKA/Migrating+to+Tika+2.0.0). However I still see `Keywords` in some of the results we get from tika. I'll look into this a little more.

Reply:
OK, it looks like some parsers generate some of the old values (like "Keywords"), and some generate the new ones (like "dc:subject"). So I changed it to check both. I double-checked all of the metadata property names, and "dc:subject" is the only one of the new names that we weren't checking.

Reply:
I believe you hit this one: https://issues.apache.org/jira/plugins/servlet/mobile#issue/TIKA-3629